build-mbl/poky-build/Dockerfile

```diff
@@ -0,0 +1,159 @@
+FROM ubuntu:18.04
+
+# if running as a jenkins slave:
```

Comment:
Why do we want jenkins logic in here?

Reply:
Removed - the Dockerfile is now based on the mbl version.